pkg/auth/authproviders/oidc/backend_impl.go

```diff
@@ -641,12 +641,13 @@ func userInfoToExternalClaims(userInfo *userInfoType) *tokens.ExternalUserClaim
 	}
 
 	// Add sso.redhat.com attributes.
-	if userInfo.OrgID != "" {
-		claim.Attributes[orgIDAttribute] = []string{userInfo.OrgID}
-	}
-	if userInfo.IsOrgAdmin {
-		claim.Attributes[authproviders.GroupsAttribute] = append(claim.Attributes[authproviders.GroupsAttribute], orgAdminGroup)
-	}
+	// no need of RHSSO attributes, using claim mappings for is_org_admin and org_id
```

Comment:
Why did you comment this out instead of just removing the code? Commented out code is a smell imo that is best avoided.

Reply:
\+